CMakeLists.txt

```diff
@@ -188,6 +188,17 @@ set (Seastar_ALLOC_PAGE_SIZE
   STRING
   "Override the Seastar allocator page size, in bytes.")
 
+function (set_option_if_package_is_found option_name package_name)
+  # if the package is found, set the option on behalf of user unless it is
+  # explicitly specified,
+  if (DEFINED ${option_name})
+    return ()
+  endif ()
+  if (${package_name}_FOUND)
+    set (${option_name} ON CACHE BOOL "")
```

Comment:
Sorry for saying this (and of course it's not relevant to your patch ;-)), but this cmake language is really ugly. This is so weird. One of the common complaints about autoconf was its weird m4 language. So cmake went ahead and invented the only language which is even weirder? :-)

Reply:
are you referencing `${package_name}_FOUND` ? or the `set()` command? i wanted to use the `package_name` as the parameter for better readability from the caller's perspective, as i think we will repeat the pattern at least for the DPDK support. regarding the latter, yeah, guess, i am used to it already =)